apps/web-giddh/src/app/vouchers/list/list.component.html

```
@@ -5969,6 +5972,71 @@ <h3>
+            </button>
+        </div>
+        <mat-dialog-content class="dialog-body font-size-15">
+            @if (isDscCertificateLoading) {
+                <div class="d-flex align-items-center justify-content-center py-3">
+                    <mat-spinner [diameter]="30"></mat-spinner>
+                </div>
+            } @else {
+                @if (dscCertificates.length === 0) {
+                    <p class="text-muted">{{ localeData?.dsc_pin_dialog?.no_certificates }}</p>
+                }
+                @if (dscCertificates.length > 0) {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)